scripts/west_commands/runners/jlink.py

```diff
@@ -318,6 +323,7 @@ def flash(self, **kwargs):
         self.logger.debug('JLink commander script:\n' +
                           '\n'.join(lines))
 
+
```

Comment:
Please don't make a habit of mixing in unrelated whitespace changes. A commit should do one thing.

Here is a short presentation on our upstreaming process which explains this and more, including the "no merge commits" rule: https://www.youtube.com/watch?v=mnVNLM7ynMQ